### PR TITLE
Update the openvpn container

### PR DESCRIPTION
**Pull request by mrIncompetent**
**What this PR does / why we need it**:
Update the openvpn container

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by mrIncompetent** [comment]
/hold 

i need to update the images

File: openshift_addons/release.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 set -ex
-export TAG=v0.6
+export TAG=v0.7-dev1
```
Comment (alvaroaleman) [inline openshift_addons/release.sh]: Why was this changed?

Comment (mrIncompetent) [reply openshift_addons/release.sh]: The openvpn container runs as client inside the user cluster. We deploy it via an Addon.
The corresponding addon manifest is changed as well: https://github.com/kubermatic/kubermatic/pull/3161/files#diff-c7c1b998591380695968958c8ad5a1ca

**Review by alvaroaleman — COMMENTED** [review]
/lgtm
/approve
